### PR TITLE
[UXE-3308] feat: add accordion to edge application rules engine criterias 

### PR DESCRIPTION
**Pull request by aloisio-m-bastian**
### What is the new behavior introduced by this PR?
Add accordion to each criteria in edge application rules engine
### Does this PR introduce breaking changes?
- [x] No
- [ ] Yes 

### Does this PR introduce UI changes? Add a video or screenshots here.
image

### Does it have a link on Figma?

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (feat, test, refactor, etc)
- [x] Application responsiveness was tested to different screen sizes
- [x] Code is formatted and linted
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

File: src/views/EdgeApplicationsRulesEngine/FormFields/FormFieldsEdgeApplicationsRulesEngine.vue [hunk 1]
```diff
@@ -593,121 +594,129 @@
...
+                    :disabled="isDefaultPhase"
+                    :data-testid="`edge-application-rule-form__criteria-operator[${criteriaIndex}][${conditionalIndex}]`"
+                  />
+                </div>
+                <div class="flex flex-col w-full">
+                  <FieldText
+                    :data-testid="`edge-application-rule-form__criteria-input-value[${criteriaIndex}][${conditionalIndex}]`"
+                    v-if="
+                      criteria[criteriaIndex].value[conditionalIndex].operator !== 'exists' &&
+                      criteria[criteriaIndex].value[conditionalIndex].operator !== 'does_not_exist'
```
Comment (william-tome) [inline src/views/EdgeApplicationsRulesEngine/FormFields/FormFieldsEdgeApplicationsRulesEngine.vue:671]: Isso poderia estar em uma função ao invés do template

Comment (aloisio-m-bastian) [reply src/views/EdgeApplicationsRulesEngine/FormFields/FormFieldsEdgeApplicationsRulesEngine.vue:671]: não tinha mexido nessa parte, apenas movi pra dentro do accordion, mas posso incluir

Comment (william-tome) [reply src/views/EdgeApplicationsRulesEngine/FormFields/FormFieldsEdgeApplicationsRulesEngine.vue:671]: Por favor.
Valeu!!

**Review by william-tome — APPROVED** [review]

File: src/views/EdgeApplicationsRulesEngine/FormFields/FormFieldsEdgeApplicationsRulesEngine.vue [hunk 2]
```diff
@@ -486,13 +488,19 @@
...
-  const getBehaviorLabel = (behaviorItem) => {
-    return behaviorItem.isFirst ? 'Then' : 'And'
+  const isNotLastCriteria = (criteriaIndex) => {
+    return criteriaIndex !== criteria.value.length - 1
+  }
+
+  const shouldRenderCriteriaValueInput = (criteriaIndex, conditionalIndex) => {
+    return (
+      criteria.value[criteriaIndex].value[conditionalIndex].operator !== 'exists' &&
+      criteria.value[criteriaIndex].value[conditionalIndex].operator !== 'does_not_exist'
```
Comment (matheus-sunderhus) [inline src/views/EdgeApplicationsRulesEngine/FormFields/FormFieldsEdgeApplicationsRulesEngine.vue:498]: here you can use the operator 'includes 'to check this condition :
```

const operatorsWithoutCriteriaValueInputs = ['exists','does_not_exist']
const {operator: criteriaOperator}  = criteria.value[criteriaIndex].value[conditionalIndex]
return  ! operatorsWithoutCriteriaValueInputs.includes(criteriaOperator)


```

**Review by matheus-sunderhus — APPROVED** [review]
minor changes, not blocking to PR

**Pull request merged**